File: ClientGz/src/app/model/bill.ts
import { BillDetail } from "./billDetail";
import { BillStatus } from "./billStatus";

export interface Bill
{
    id: number;

    name: string;
    phone: string;
    address: string;
    totalPrice:number;
    orderDate : Date
    billStatus : BillStatus;
    billDetail:BillDetail[];
}

File: ServerGz/Models/Bill.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System;
namespace ServerGz.Models
{
    public class Bill
    {

        public int id { get; set; }

        public string name { get; set; }
        public string phone { get; set; }
        public string address { get; set; }

        public double totalPrice { get; set; }
        public DateTime orderDate { get; set; }

        public string accountName { get; set; }

        [ForeignKey("accountName")]
        public Account account { get; set; }
        public BillStatus billStatus { get; set; }
        public ICollection<BillDetail> billDetail { get; set; }
    }
}
